plugins/woocommerce/changelog/48199-fix-make-sure-element-exist-for-customerscore

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+</details>  <details>  <summary>Changelog Entry Comment</summary>  Ensures that the root element exists before attempting to render the CustomerEffortScoreTracksContainer component.
```

Comment:
```suggestion
Ensures that the root element exists before attempting to render the CustomerEffortScoreTracksContainer component.
```

Reply:
Thanks @chihsuan again 👍 

I'm keep making the same mistake 😞